### PR TITLE
mgr/DaemonServer: try python handlers if command not found

### PR DESCRIPTION
**Pull request by tserong**
If the command invoked wasn't in mgr_commands, handle_command()
would immediately return -EINVAL rather than eventually falling
back to trying the registered python handlers.

Signed-off-by: Tim Serong <tserong@suse.com>

### CODE REVIEW

File: src/mgr/DaemonServer.cc [hunk 1]
```diff
@@ -628,35 +653,7 @@ bool DaemonServer::handle_command(MCommand *m)
...
-    // FIXME: go run this python part in another thread, not inline
-    // with a ms_dispatch, so that the python part can block if it
-    // wants to.
-    dout(4) << "passing through " << cmdmap.size() << dendl;
-    stringstream ds;
-    r = handler->handle_command(cmdmap, &ds, &ss);
-    odata.append(ds);
-    return _reply(m, 0, ss.str(), odata);
-  }
+  return true;
```
Comment (tchaikov) [inline src/mgr/DaemonServer.cc]: might want to print an error message or even assert here? if we claim to support this command, but failed to handle it. that would be a bug, i believe.

**Comment by jcsp** [comment]
This is already fixed in https://github.com/ceph/ceph/pull/14578, although I'm confused right now because I can see the DaemonServer.cc changes in the "Files changed" part of that PR but not in the list of commits!

**Comment by jcsp** [comment]
Ah, I had screwed up a rebase on that other one.  Fixed it now, the fix for handle_command is https://github.com/ceph/ceph/pull/14578/commits/6f038eba89abec4b951b724a66a3e130a0142c9c

**Comment by tserong** [comment]
> Fixed it now, the fix for handle_command is 6f038eb

That commit seems to have vanished, looks like d22c0ad now.

File: src/mgr/DaemonServer.cc (hunk 1 above)
Comment (tserong) [inline src/mgr/DaemonServer.cc]: I wasn't sure of the exact semantics, whether returning true/false here meant the individual  command was handled, or whether handle_command() was just meant to return true to indicate that MSG_COMMAND was processed somehow.

Anyway, as @jcsp has fixed this separately (and probably better) in #14578, may as well close this PR.

**Pull request closed without merging**